snac-cooperative/snac-laravel
# [#34722] Add Authentication and Authorization to api routes

**Link to Ticket:**  
https://app.shortcut.com/snac/story/34722/require-user-to-be-an-authenticated-editor-for-any-post-put-delete-actions-on-concept-vocabulary

**PR Summary:**  
Sets up authentication and authorization for api routes. Also adds in `artisan user:add` helper command

**Testing Instructions:**  
1. Confirm that user must be logged in to hit protected create, update, delete api routes
2. Feature tests pass when running `ddev artisan test`
3. `artisan user:add` can be used to interactively add a user to the system

**Notes:**  
- test_authorized_user_can_update_concept_relationships will fail due to issue with concept relationships in the concept model
